src/Admin/BaseFieldDescription.php

```diff
@@ -124,6 +124,11 @@
      */
     protected $help;
 
+    /**
+     * @var array cached object field getters
```

Comment:
If array of strings, please use `string[]`, and so on

Reply:
It's not `string[]`. The structure is
```php
self::$fieldGetters['some_string_key'] = [
    'method' => 'getter', // | 'call' | 'var'
    'getter' => 'getVarValueMethodName' // if not getter method, not set
];
```

Reply:
Then it should be `array[]`